go/oasis-node/cmd/registry/node/node.go

```diff
@@ -124,26 +120,9 @@ func doInit(cmd *cobra.Command, args []string) { // nolint: gocyclo
 			os.Exit(1)
 		}
 		logger.Info("entity ID provided, assuming self-signed node registrations")
-
-		isSelfSigned = true
 	} else {
-		entity, entitySigner, err = cmdCommon.LoadEntitySigner()
-		if err != nil {
-			logger.Error("failed to load entity and its signer",
-				"err", err,
-			)
-			os.Exit(1)
-		}
-
-		entityID = entity.ID
-		isSelfSigned = !entity.AllowEntitySignedNodes
-		if viper.GetBool(CfgSelfSigned) {
-			isSelfSigned = true
-		}
-		if !cmdFlags.DebugDontBlameOasis() && !isSelfSigned {
-			logger.Error("entity signed node descriptors are prohibited")
-			os.Exit(1)
-		}
+		logger.Error("entity-signed node descriptors are prohibited")
+		os.Exit(1)
 		defer entitySigner.Reset()
```

Comment:
This should probably be removed as well as it is unreachable (and would crash if reached)?

Reply:
Good catch, fixed! :)